Chat/Chat.cpp

```diff
@@ -0,0 +1,168 @@
+#include "Chat.h"
+#include "Network.h"
+#include <iostream>
+#include <thread>
+#include <chrono>
+
+#include <conio.h>
+
+using std::cout;
+using std::cin;
+using std::endl;
+
+
+Chat::Chat(){}
```

Comment:
Codestyle warning. You can write brackets as here or as second case in earlier comment. But you should choose one style and write it everywhere.